src/components/questions/IntegerQuestion.vue

```diff
@@ -167,6 +167,9 @@ export default {
       }
     },
     question() {
+      if (this.selected === "") {
+        document.getElementById("integerInput").value = "";
```

Comment:
This seems redundant, isn't the value of the input element `""` if `this.selected` equals `""` because of the `v-model`? Also, if this is needed we should use [$refs](https://vuejs.org/v2/guide/components-edge-cases.html?#Accessing-Child-Component-Instances-amp-Child-Elements) instead of `document.getElementById`.

Reply:
yes, selected is "" if an invalid number was entered. problem was, that the input itself, does not clear the input. Instead it still shows the invalid number. Problem gets worse if several Integer-questions a stacked together. The questions change, but the input is still the same and displays the same invalid number. To fix it, i set the input to "" if selected/v-model is ""